fiftyone/core/dataset.py

```diff
@@ -123,7 +130,7 @@ def _validate_dataset_name(name, skip=None):
     return slug
 
 
-def load_dataset(name):
+def load_dataset(name, create_if_missing=False):
```

Comment:
@minhtuev I'm onboard with this new parameter 👍 

Can we call it `create_if_necessary` for consistency with the nomenclature used in other places, such as `load_zoo_dataset(name, download_if_necessary=True)`? 🤓

Reply:
Done!